client/state/ui/selectors/site-data.ts

```diff
@@ -8,6 +8,7 @@ export interface SiteData {
 	domain: string;
 	locale: string;
 	options?: SiteDataOptions;
+	wpcom_url?: string;
```

Comment:
@jdc91 What do we add this for? I'm asking because I don't see any related code logic.

Reply:
This is to preload the url to the domain picker.